guides/partials.md

```diff
@@ -1,36 +1,54 @@
 ---
-title: DustJS | Getting Started
+title: DustJS | Partials
 layout: guides
 permalink: /guides/partials/
 ---
 
-## Partials
+# Partials
 
-A Dust partial promotes "DRY" behavior for reusable/shared markup. Partials themselves can be composed of one or more partials. Partials rely on the JSON/context of the parent template invoking it. Lets peek under the covers to see how a Dust partial works.
+A Dust partial promotes *DRY* ("don't repeat yourself") conventions in shared markup. Breaking templates up into partials can make them easier to maintain and reuse, and partials themselves can be composed of one or more partials. This is the basis for components or reusable templates for tasks like including a common header and footer on multiple pages. Lets peek under the covers to see how a Dust partial works.
 
-You can have multiple .dust files and reference one Dust template as part of another one. This is the basis for components or reusable templates for tasks like a common header and footer on multiple pages. Note that the .dust file extension is used here in examples, but .tl is also commonly seen. Since it only matters to the build process you can use whatever extension works for you.
+A partial template is added to another template using Dust's partial syntax. Note that the tag is self-closing (`/}`).
 
-Let's see how the Dust template rendering knows about a template. As we said earlier, Dust templates are compiled to JavaScript. Part of that compiled result is a call to `dust.register(name, functionImplementingCompiledTemplate)`.
+```
+{>"template-name"/}
+```
 
-Like sections, partials accept parameters so you can build reusable components that are parameterizable easily. This gives you the same foundation for building libraries as other languages. By passing all the data into the partial using parameters, you isolate the partial from any dependence on the context when it is invoked. So you might have things like `{>header mode="classic" /}` to control the header behavior.
+Partials rely on the JSON context of the parent template invoking it. Like [sections](/guides/getting-started/#sections), partials also accept parameters that add extra references. By passing more data into the partial using parameters, you can build components that are customized to different contexts easily. For example, there might be several versions of a header that you wish to include, which can be controlled with code like `{>"header" mode="classic"/}`.
```

Comment:
"For example, there might be several versions of a header that you wish to include, which can be controlled using parameters: `{>"header" mode="classic"/}`."